types/next-auth/client.d.ts

```diff
@@ -0,0 +1,105 @@
+import { FC } from 'react';
+
+interface Session {
+    user: {
+        name: string;
+        email: string;
+        image: string;
+    };
+    accessToken: string;
```

Comment:
Access token is not always available on the session object – only if `session.jwt` is set to `false` on the configuration. So this should probably be:

```suggestion
    accessToken?: string;
```

Reply:
https://github.com/nextauthjs/next-auth/blob/2ea64045cb28cbe0167a9bd8cd65c2d752e452fb/src/server/routes/session.js#L19